k8s/sonarqube/README.md

```diff
@@ -0,0 +1,392 @@
+```
+
+Generate random password for PosgreSQL:
+
+```shell
+export POSTGRESQL_DB_PASSWORD=$(openssl rand 9 | openssl base64 -A | openssl base64)
+```
+
+Enable Stackdriver Metrics Exporter:
+
+> **NOTE:** Your GCP project should have Stackdriver enabled. For non-GCP clusters, export of metrics to Stackdriver is not supported yet.
+
+By default the integration is disabled. To enable, change the value to `true`.
+
+```shell
+export METRICS_EXPORTER_ENABLED=false
+```
+
+#### Expand the manifest template
+
+Use `helm template` to expand the template. We recommend that you save the
+expanded manifest file for future updates to the application.
+
+```shell
+helm template chart/wordpress \
+  --name="$APP_INSTANCE_NAME" \
+  --namespace="$NAMESPACE" \
+  --set sonarqube.image=$IMAGE_SONARQUBE \
+  --set postgresql.image=$IMAGE_POSTGRESQL \
+  --set postgresql.exporter.image=$IMAGE_POSTGRESQL_EXPORTER \
+  --set postgresql.db.password=$DB_PASSWORD  \
+  --set metrics.image=$METRICS_EXPORTER_ENABLED > ${APP_INSTANCE_NAME}_manifest.yaml
+```
+#### Apply the manifest to your Kubernetes cluster
+
+Use `kubectl` to apply the manifest to your Kubernetes cluster. This installation will create:
+
+- An Application resource, which collects all the deployment resources into one logical entity
+- A ServiceAccount for the SonarQube and PostgreSQL Pod.
+- A PersistentVolume and PersistentVolumeClaim for SonarQube and PostgreSQL. Note that the volumes isn't be deleted with application. If you delete the installation and recreate it with the same name, the new installation uses the same PersistentVolumes. As a result, there is no new database initialization, and no new password is set.
+- A Secret with the PostgreSQL initial random password
+- A Deployment with SonarQube and PostgreSQL.
+- A Service, which exposes PostgreSQL and SonarQube to usage in cluster.
+
+
+```shell
+kubectl apply -f "${APP_INSTANCE_NAME}_manifest.yaml" --namespace "${NAMESPACE}"
+```
+
+#### View the app in the Google Cloud Console
+
+To get the Console URL for your app, run the following command:
+
+```shell
+echo "https://console.cloud.google.com/kubernetes/application/${ZONE}/${CLUSTER}/${NAMESPACE}/${APP_INSTANCE_NAME}"
+```
+
+To view the app, open the URL in your browser.
+
+# Using the SonarQube community edition
+
+#### View the application in the Google Cloud Console
+
+To get the Console URL for your application, run the following command:
+
+```shell
+echo "https://console.cloud.google.com/kubernetes/application/${ZONE}/${CLUSTER}/${NAMESPACE}/${APP_INSTANCE_NAME}"
+```
+Application does not exposed to external world. To get access to SonarQube run the following command:
+```bash
+    kubectl port-forward \
+      --namespace $NAMESPACE \
+      svc/$APP_INSTANCE_NAME-sonarqube-svc \
+      9000:9000
+```  
+SonarQube will available on `localhost:9000`. All interaction with application goes thru `9000` port. Cli also will be available.    
```

Comment:
Please change "All interaction with application goes thru `9000` port. Cli also will be available."
to
"You can use web browser to connect to SonarQube HTTP server, or you can use `???` CLI tool to directly use SonarQube API."

??? should be the name of cli tool